<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MLClass</class>
 <widget class="QMainWindow" name="MLClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ML</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tblData">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>471</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="wColFilter" native="true">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>20</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QLabel" name="lblColFil">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Columns Filter:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="wTools" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>431</width>
      <height>81</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,0,4,0,1,0">
     <item>
      <widget class="QWidget" name="w1" native="true">
       <property name="minimumSize">
        <size>
         <width>108</width>
         <height>79</height>
        </size>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>93</width>
          <height>56</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QPushButton" name="btnData">
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>Import Data</string>
         </property>
        </widget>
        <widget class="QPushButton" name="btnTrain">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="text">
          <string>Train</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="wChoose" native="true">
       <property name="minimumSize">
        <size>
         <width>157</width>
         <height>79</height>
        </size>
       </property>
       <widget class="QSplitter" name="splitter_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>148</width>
          <height>51</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLabel" name="lblMerge">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Choose column to merge:</string>
         </property>
        </widget>
        <widget class="QComboBox" name="cbCommon">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <item>
          <property name="text">
           <string>--Not Selected--</string>
          </property>
         </item>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="cbxFilter">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Filter rows
with null values</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ML.qrc"/>
 </resources>
 <connections/>
</ui>
